scenarios/multi_pow_01/reports/Multi-PoW Mining Simulation Approach.md

```diff
@@ -0,0 +1,85 @@
+# Multi-PoW Mining Simulation Approach
+
+(_The code base is available on GitHub at [**tari-labs/modelling/scenarios/multi_pow_01**](https://github.com/tari-labs/modelling/tree/master/scenarios/multi_pow_01)_)
+
+## Mining
+
+To simulate a constant probability of producing a block at the correct difficulty, an oracle is used that asks each algorithm to produce the next block. Each algorithm runs independently, without any inter-dependencies, using its own difficulty adjustment. Mining is based on the average time it takes to solve a block for a given hash rate and target difficulty (see _**Block Time Estimate**_).
```

Comment:
I think someone may ask "why did you not just use the probabilities themselves to determine the next block?"
e.g. you could have had a loop every second and determined the probability for each algo to find a block

Reply:
Done